File: cpp/Estruturas condicionais/notas.cpp
#include <iostream>
using namespace std; 
int main() {

double nota1, nota2, notaFinal;

cout << "Digite a primeira nota:" << endl;
cin >> nota1;

cout << "Digite a segunda nota:" << endl;
cin >> nota2;

    notaFinal = (nota1 + nota2) / 2;


if (notaFinal < 5) { 
   cout << "Reprovado com nota " << notaFinal;
}

else if (notaFinal < 7 ) {
    cout << "Voce esta de recuperacao com nota " << notaFinal;
}

else {
    cout << "Aprovado com nota " << notaFinal;
}


    return 0;
}

File: java/Estruturas condicionais/Notas.java
import java.util.Locale;
import java.util.Scanner;
public class Notas {
     public static void main(String[] args) { 
 Locale.setDefault(Locale.US);
 Scanner sc = new Scanner(System.in);

double A, B, media;

System.out.println("Digite a primeira nota");
A = sc.nextDouble();

System.out.println("Digite a primeira nota");
B = sc.nextDouble();

media = (A + B) / 2;

if (media < 4) {

System.out.println("Voce esta reprovado. Sua media é: " + media);
}

else if (media > 6) {
    System.out.println("Voce esta aprovado! Sua media e " + media);
}

else {
    System.out.println("Voce esta de recuperacao. Sua media e " + media);
}





 sc.close();
 }
}
